pages/timer/[id].js

```diff
@@ -0,0 +1,85 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import LayoutSytle from "../../components/LayoutStyle";
+
+export default function Timer({ tasks }) {
+  const router = useRouter();
+  const { id } = router.query;
+  const [durationClock, setDurationClock] = useState(null);
+
+  useEffect(() => {
+    setDurationClock({ sec: duration, start: true });
+  }, []);
+
+  useEffect(() => {
+    let timer;
+    if (durationClock !== null) {
+      timer = setInterval(() => {
+        if (durationClock.sec > 0) {
```

Comment:
also else if here I think

Reply:
@Mark-Krensel Thank you very much for your suggestions! I did some code refactoring. Would you like to have a look again?